ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1689,3 +1689,10 @@ FCU_DSP_DFVSC_DTM_RTM:
   - DTM
   - RTM
 
+FCU_DFMSC_ZTC_RMM_FDPSM:
```

Comment:
Does the unit have `DXZC`, `DXZTC`, `CHWZTC` or `CHWZC`?

Reply:
no it doesn't

Reply:
Does it have both heating and cooling?

Reply:
@kevin-hereworks it would be odd to see a FCU without heating or cooling. I notice that this is a VRF system, so are you trying to model the indoor units here?

Reply:
yes the indoor units, they have heating and cooling too